### DISCUSSION TITLE
Production and title keys

### DISCUSSION THREAD

**Discussion by Giacomo121097** [skyline-emu/skyline#1446] [Game Requests]
![Screenshot_2022-06-02-13-07-13-077_skyline emu](https://user-images.githubusercontent.com/106743298/171617755-eedfa0f3-507a-483b-84a4-6f012eca3808.jpg)

I Need of production keys and title keys, where i can find?
Please i Need help

**Comment by Swizzlefowl** [comment]
You have to dump them from your own switch.

**Reply by Giacomo121097** [reply]
I have a switch but i don't know where are the keys

**Reply by Giacomo121097** [reply]
And i don't know how to download

**Reply by bylaws** [reply]
https://yuzu-emu.org/help/quickstart/

**Reply by Giacomo121097** [reply]
Thanks, but i haven't PC, can i do it Just with a phone?🤔
